finance/portfolio-performance/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           java 1.0
+
+github.setup        buchen portfolio 0.50.3
+name                portfolio-performance
+categories          finance
+platforms           darwin
+license             EPL-1
+maintainers         {@janosch mailbox.org:janosch1} openmaintainer
+description         A simple tool to calculate the overall performance of an investment portfolio.
+long_description    An open source tool to calculate the overall performance of an investment portfolio \
+                    - across all accounts - using True-Time Weighted Return or Internal Rate of Return.
+homepage            https://www.portfolio-performance.info/en/
+
+supported_archs     x86_64
+java.version        11+
+depends_build-append port:maven3
+use_configure       no
+
+# The tycho version used in PP leads to build failures. Using version 2.0.0 instead of 1.5.1 fixes it.
+patchfiles          patch-increase-tycho-version.diff \
+                    patch-disable-autoupdate-pref.diff \
+                    patch-remove-autoupdate-dialog.diff
+
+set maven_local_repository ${worksrcpath}/.m2/repository
+pre-build {
+    file mkdir ${maven_local_repository}
+}
+
+build.cmd           mvn3
+build.pre_args      clean verify -f portfolio-app -Ppackage-distro \
+                    -Dmaven.repo.local=${maven_local_repository} \
+                    -DskipTests
+
+destroot            {
+                      copy -- ${worksrcpath}/portfolio-product/target/products/name.abuchen.portfolio.product/macosx/cocoa/x86_64/PortfolioPerformance.app \
+                      ${destroot}${applications_dir}
+                    }
```

Comment:
The spacing is odd here. The braces don't need to follow the mode line. Also, the copy command should only be indented out 4 spaces.

Reply:
Thanks for the hint. It is fixed now.